Lombiq.Tests.UI.Samples/Tests/BasicOrchardFeaturesTests.cs

```diff
@@ -13,18 +14,40 @@ namespace Lombiq.Tests.UI.Samples.Tests;
 // the whole test suite.
 public class BasicOrchardFeaturesTests : UITestBase
 {
+    // We could reuse the previously specified SetupHelpers.RecipeId const here but it's actually a different recipe for
+    // these tests.
+    private const string BasicOrchardFeaturesTestsRecipeId = "Lombiq.OSOCE.BasicOrchardFeaturesTests";
+
     public BasicOrchardFeaturesTests(ITestOutputHelper testOutputHelper)
         : base(testOutputHelper)
     {
     }
 
-    // We could reuse the previously specified SetupHelpers.RecipeId const here but it's actually a different recipe for
-    // this tests.
     [Theory, Chrome]
     public Task BasicOrchardFeaturesShouldWork(Browser browser) =>
         ExecuteTestAsync(
-            context => context.TestBasicOrchardFeaturesAsync("Lombiq.OSOCE.BasicOrchardFeaturesTests"),
+            context => context.TestBasicOrchardFeaturesAsync(BasicOrchardFeaturesTestsRecipeId),
             browser);
+
+    // For testing, we can use the already existing databases. Here, we set up the application, then we take a snapshot
+    // of it, then we use the "ExecuteTestFromExistingDBAsync()" to run the test on that. Then we test the basic Orchard
+    // features as we did above.
+    [Theory, Chrome]
+    public Task BasicOrchardFeaturesShouldWorkWithExistingDBSetup(Browser browser) =>
```

Comment:
I put this test here (`BasicOrchardFeaturesTests.cs`), because I'm using the `TestBasicOrchardFeaturesExceptSetupAsync()` method inside the test, so I thought this would fit here. 

I'm using that method because we need some tests to do inside `ExecuteTestFromExistingDBAsync()`, and it's better to check that everything works correctly from the previously created snapshot.

Reply:
It would be better in a new class, inserted into the series of tutorials, because using an existing DB snapshot is not really something most users would need.

Reply:
Okay, I will move it then.